<h1>
 Title: Test Cases are Passing but Query_Roster Problem
</h1>
<h3>
 Author: Anonymous
</h3>
<h3>
 Date: 2021-03-09T03:36:16.385042+11:00
</h3>
<h3>
 Category: Assignments
</h3>
<h3>
 Vote Count: 0
</h3>
<document version="2.0">
 <paragraph>
  For a1_second, all my test cases for the functions have run and passed. However, when I try to run query_roster on Python, I get back a ValueError that states "Substrings not found" for several lines of my code. Is this a problem with my function body or names?
 </paragraph>
 <paragraph>
 </paragraph>
</document>
<h3>
 Author: Dylan Castillo (None)
</h3>
<h3>
 Vote Count: 0
</h3>
<document version="1.0">
 <paragraph>
  Could you post a screenshot?
 </paragraph>
</document>
<h3>
 ------------------------------------
</h3>
<h3>
 Author: Anonymous
</h3>
<h3>
 Vote Count: 0
</h3>
<document version="2.0">
 <paragraph>
  I covered the functions, but when I had originally tested them using a1_first.py, they all passed
 </paragraph>
 <figure>
  <image height="118.49056603773586" src="https://static.us.edusercontent.com/files/GdoOcoBlFcaA9WIRMABVje8i" width="520"/>
 </figure>
</document>
<h3>
 ------------------------------------
</h3>
<h3>
 Author: Prof. Lee (she/they) (admin)
</h3>
<h3>
 Vote Count: 0
</h3>
<document version="2.0">
 <paragraph>
  Are the function names that you have redacted helper functions you have created, or functions whose headers we gave you?
  <break>
  </break>
  <break>
  </break>
  If the latter, please post an unredacted screenshot (it's OK to post error messages in that case)
 </paragraph>
 <paragraph>
 </paragraph>
</document>
<h3>
 ------------------------------------
</h3>
<h3>
 ----------- REPLIES -----------
</h3>
<h3>
 Author: Prof. Lee (she/they) (admin)
</h3>
<h3>
 Vote Count: 0
</h3>
<document version="2.0">
 <paragraph>
  (adding an "answer" for now, as we didn't hear back from the original poster (OP))
 </paragraph>
</document>
<h3>
 ------------------------------------
</h3>